generators/common/base-generator.d.ts

```
@@ -0,0 +1,43 @@
+    generateMainJavaCode(configOptions: ConfigOptions, templates: Array<string | Template>): void;
+    generateMainResCode(configOptions: ConfigOptions, templates: Array<string | Template>): void;
+    generateTestJavaCode(configOptions: ConfigOptions, templates: Array<string | Template>): void;
+    generateTestResCode(configOptions: ConfigOptions, templates: Array<string | Template>): void;
+    generateFiles(configOptions: ConfigOptions, templates: Array<string | Template>, srcRoot: string, baseFolder: string): void;
+    _generateCode(configOptions: ConfigOptions, templates: Array<string | Template>, srcRoot: string, baseFolder: string, packageFolder: string): void;
+    _formatCode(configOptions: ConfigOptions, baseDir: string | null): void;
+    _formatCodeMaven(configOptions: ConfigOptions, baseDir: string | null): void;
+    _formatCodeGradle(configOptions: ConfigOptions, baseDir: string | null): void;
+    _isWin(): boolean;
+}
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)